INSTALL.md

```diff
@@ -0,0 +1,56 @@
+## Install Instructions
+
+### 4.20
+
+
+
+* Download the entire project from github via git or zip download.
+* If there is a `ComputeShaderDev` folder in `Plugins` folder, remove it (otherwise it will most likely not work)
+* Open `OceanProject.uproject` with **UE 4.20** (It's only tested on **4.20.3**).
+
+* The project should load just fine.
+
+* Follow [this guide from the wiki](https://github.com/UE4-OceanProject/OceanProject/wiki/How-to-add-to-your-project) to install into your own project.
+
+
+#### How to package (and include OceanPlugin in the build)
+
+* Close your project and UE4
+
+* Move the `OceanPlugin` folder (which is on `Plugins` folder) to `Program Files/Epic Games/UE_4.20/Engine/Plugins/Runtime`
+
+* Edit your `.uproject` and add this:
```

Comment:
I'm confused here, should we have a single .uproject file that can work in the editor and for packaging? A default standard would be approved here. See comment in the .uproject file

Reply:
Perfect